frontend/components/etapes-formulaire/Etape3.vue

```diff
@@ -120,6 +160,46 @@ const downloadLettreInfo = () => {
   window.open(getLettreInfoUrl(store.currentId), '_blank')
 }
 
+const isEmailValid = computed(() => {
+  if (email.value.trim() === '') return false
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
+  return emailRegex.test(email.value)
+})
+
+const emailInputDescribedBy = computed(() => {
+  if (emailError.value) return errorId
+  if (emailSuccess.value) return successId
+  return null
+})
+
+const clearEmailSuccessError = () => {
+  emailError.value = ''
+  emailSuccess.value = ''
+}
+
+const sendEmail = async () => {
+  if (!isEmailValid) {
```

Comment:
```suggestion
  if (!isEmailValid.value) {
```
pour la valeur, pas la ref